journald/journald.hpp

```diff
@@ -32,6 +37,19 @@ struct Flags : public virtual flags::FlagsBase
       "before it is written to journald.  See '--labels'."
       "\n");
 
+    add(&Flags::destination_type,
+        "destination_type",
+        "Determines where logs should be piped.\n"
+        "Valid destinations include: 'journald', 'sandbox', or 'both'.",
```

Comment:
1. "both" is vague and not future-proof in case we add more types later (syslog, sandbox-without-logrotate); please either rename it to something like "journald_and_sandbox" or turn it into a `,`-delimited list like "journald,sandbox" and we can check for set.contains() when choosing when to apply a strategy's behavior.
2. "sandbox" seems like a misnomer, since you actually copied the `LogrotateLoggerProcess`, and there's a separate `SandboxContainerLoggerProcess` that doesn't do log-rotation.